ocaml/utils/compilation_unit.ml

```diff
@@ -51,6 +46,10 @@ module Name : sig
 
   val to_string : t -> string
 
+  val of_head_of_global_name : Global_module.Name.t -> t
```

Comment:
I'm sad that `utils` refers to `typing` - let's talk about this

Reply:
This has been true for a while now, though, with `Compilation_unit` and `Symbol` both referring to `Ident` but being broader in scope than just typing.